winui/Avatar-view/visual-styles.md

```diff
@@ -1,10 +1,404 @@
 ---
 layout: post
 title: Visual Styles in WinUI AvatarView control | Syncfusion
-description: This section will explain about the different types of visual styles that exists in WinUI AvatarView.
+description: This section will provide an explanation of the various visual styles that are available in WinUI AvatarView.
 platform: WinUI
 control: AvatarView
 documentation: ug
 ---
 
-# Visual Styles in WinUI AvatarView
\ No newline at end of file
+# Visual Styles in WinUI AvatarView
+
+`AvatarView` control allows for customization of visual styles by providing a range of built-in styles. You can configure the visual style of your AvatarView by using the `AvatarShape` property and select from the available options, such as:
+
+* Custom
+* Circle
+* Square
+
+## Custom
+
+Custom type enables extensive customization options such as adjusting the size, font size, corner radius, and other properties.
+
+## Circle
+
+The `AvatarView` control allows you to set the size of the circle by utilizing the predefined `AvatarSize` values such as:
+
+* ExtraLarge
+* Large
+* Medium
+* Small
+* ExtraSmall
+
+The following code sample illustrates how to set the visual styles of AvatarView to `Circle`.
+
+{% tabs %}
+{% highlight xaml %}
+
+<Page.Resources>
+    <ResourceDictionary>
+    <Style x:Key="AvatarViewStyle"  TargetType="syncfusion:SfAvatarView">
+        <Setter Property="ContentType" Value="CustomImage"/>
+        <Setter Property="ImageSource" Value="person.png"/>
+    </Style>
+    </ResourceDictionary>
+</Page.Resources>
+
+<Page.Content>
+    <StackPanel Orientation="Vertical" HorizontalAlignment="Stretch" VerticalAlignment="Stretch">
+        <Grid >
+                <Grid.RowDefinitions>
+                    <RowDefinition Height="*"/>
+                    <RowDefinition Height="*"/>
+                </Grid.RowDefinitions>
+                <Grid.ColumnDefinitions>
+                    <ColumnDefinition Width="*"/>
+                    <ColumnDefinition Width="*"/>
+                    <ColumnDefinition Width="*"/>
+                    <ColumnDefinition Width="*"/>
+                    <ColumnDefinition Width="*"/>
+                </Grid.ColumnDefinitions>
+
+                <syncfusion:SfAvatarView AvatarShape="Circle" AvatarSize="ExtraLarge" Grid.Row="0" Grid.Column="4" Style="{StaticResource AvatarViewStyle}"/>
+                <TextBlock Text="ExtraLargeCircle" FontWeight="Bold" Grid.Row="1" HorizontalAlignment="Center" VerticalAlignment="Center" HorizontalTextAlignment="Center" TextAlignment="Center" Grid.Column="4" FontSize="10"/>
```

Comment:
@MadhumithanJayapathi Remove the unwanted declared in the code example.

Reply:
@Manivannan-E Removed the unwanted declared in the code example.